### ISSUE TITLE
D2 trained on some custom dataset?

### ISSUE THREAD

**Issue by jinfagang** [rwightman/efficientdet-pytorch#36]
Since coco really takes a long time to train, did u consider train D2 with a smaller dataset?

Also, I believe yolov5 repo can easily get AP up to 40+ on coco now.

**Comment by rwightman** [comment]
Someone reported doing well in a competition with the D2 on a custom dataset, it works. Very minimal tweaks to the train script if you have annotations in coco format. The Kaggle example linked on the README shows a clean usage of a completely custom dataset and albumentations aug pipeline.

Yolov5 is good, lots of other detection models can pass 40 too. Tradeoffs for all of them. The EfficientDet D2 is roughly 1/3 the param count and 1/4 the FLOPS of Yolov5-m, yet EffDet is slower on GPU due to the depthwise separable convs... so always tradeoffs, tradeoffs. I'd point out the latest Yolos are using mosaic aug and use giou or ciou in the loss. Mosaic + giou/ciou/diou could add gains here.... the augmentation currently being used here is extremely simply.
